src/actions.py

```diff
@@ -36,7 +36,7 @@ def on_get_admin_password(self, event: ops.ActionEvent) -> None:
             event: The event fired from get-admin-password action.
         """
         container = self.charm.unit.get_container(JENKINS_SERVICE_NAME)
```

Comment:
Out of the scope of this PR but I think this can now be refactored in charm init as per [this discourse post](https://discourse.charmhub.io/t/a-suggested-way-to-organise-the-code-inside-the-charm-py-file/13203/4) on organising `charm.py`

Reply:
Let's keep that in mind